client/components/dot-pager/style.scss

```diff
@@ -52,8 +52,12 @@
 	}
 
 	button {
-		width: 6px;
-		height: 6px;
+		display: inline-flex;
+
+		// Size the dots
+		min-width: 6px;
+		min-height: 6px;
```

Comment:
Maybe we can add className to dots? ex: `dot-pager__control-dots`

Reply:
I don't think adding dots to the element Is a good idea, because they're only dots because we're styling them that way. We might make them look like traditional buttons or tabs or something.

I like factoring those styles out, though, rather than applying it and then overriding it for the next/previous.